### DISCUSSION TITLE
Deduplication code is giving error.

### DISCUSSION THREAD

**Discussion by ZubeenK** [milvus-io/milvus#23279] [Q&A and General discussion]
In the example applications of Milvus, there is an application of deduplication, which can be found here: https://github.com/towhee-io/examples/blob/main/image/image_deduplication/image_deduplication.ipynb

I tried to reuse this code, but it's giving me an error. I am posting an image with the error:
![image](https://user-images.githubusercontent.com/73072849/230551849-535bdc87-4318-474e-8df4-715706e848be.png)

I have towhee version: 0.9.0 installed in my environment. Can someone help me with this error?

**Comment by shiyu22** [comment]
@ZubeenK Hi, can you check with [this issue](https://github.com/towhee-io/examples/issues/185)?
